logicrunner/builtin/contract/deposit/deposit.go

```diff
@@ -75,14 +75,8 @@ func New(migrationDaemonConfirms map[insolar.Reference]bool, txHash string, amou
 }
 
 // MapMarshal gets deposit information.
-func (d *Deposit) MapMarshal() (map[string]string, error) {
-	return map[string]string{
-		"timestamp":       d.Timestamp.String(),
-		"holdReleaseDate": d.HoldReleaseDate.String(),
-		"amount":          d.Amount,
-		"bonus":           d.Bonus,
-		"txId":            d.TxHash,
-	}, nil
+func (d *Deposit) Itself() (Deposit, error) {
```

Comment:
Why not `Get`?

Reply:
Because get what?

Reply:
Mby GetItself? But we can avoid get I think.

Reply:
But I thing Itself() is better